### DISCUSSION TITLE
What food do you like the best in your hometown

### DISCUSSION THREAD

**Discussion by i16nakatani** [NAIST-SE/AttractingContributors#865] [2023 Attraction Contributions]
I have recently become addicted to cooking and would like to try my hand at cooking a variety of dishes.
Hence, I' like you to tell me your favorite dish in your hometown, please add your idea to my README.md

Help me plz!

Please come to my fork:
https://github.com/i16nakatani/AttractingContributors-dish/blob/main/README.md

**Comment by WEI44ZHEYUAN** [comment]
I added
Please help me
https://github.com/WEI44ZHEYUAN/AttractingContributors_Songs
